pkgs/by-name/peertube-plugin-livechat/package.nix

```diff
@@ -202,8 +202,53 @@ let
       inherit (commonMeta) maintainers platforms;
     };
   };
+
+  livechatProsody = prosody.override {
+    withExtraLuaPackages = (
+      p: [
+        # Needed by one of peertube-livechat's prosody modules
+        (p.callPackage (
+          {
+            buildLuarocksPackage,
+            fetchFromGitHub,
+            fetchurl,
+            luaOlder,
+            oniguruma,
+          }:
+          buildLuarocksPackage {
+            pname = "lrexlib-oniguruma";
```

Comment:
I'd add a TODO with the Nixpkgs PR link here, so we don't forget to switch to it afterwards, but this looks good otherwise.